php/commands/src/CLI_Command.php

```diff
@@ -319,6 +305,10 @@ public function update( $_, $assoc_args ) {
 		} else {
 			EE::error( "md5 hash for download ({$md5_file}) is different than the release hash ({$release_hash})." );
 		}
+
+		$cache = EE::get_cache();
+		$cache->write( 'migrated', '0' );
```

Comment:
This should be removed now that we are handling it in the options table.

Reply:
Yes. Removed.